---
content_type: page
description: This section provides an overview of the three lab modules, information
  on working in the lab and communicating your work, pre-lab lectures, and lab materials.
draft: false
learning_resource_types:
- Labs
ocw_type: CourseSection
title: Labs
uid: c810141c-0282-3f29-da2a-83f1fe93dcb1
video_files:
  video_thumbnail_file: null
video_metadata:
  youtube_id: null
---
Contents:

- [Lab Basics](#Basics)
- [First Day: Lab Orientation](#Orientation)
- [Module 1: RNA Engineering](#RNA_Engineering)
- [Module 2: Protein Engineering](#Protein_Engineering)
- [Module 3: Cell-Biomaterial Engineering](#Biomaterial_Engineering)

Lab work is divided into three modules of eight sessions each. Learning materials for each lab session (linked below under 'Lab Materials') include an introduction, experimental protocol, and a reagents list, followed by a "for next time" homework assignment. A short pre-lab lecture preceeds most of the lab sessions.

Selected results from some labs are included courtesy of the students and used with permission.

## {{< anchor "Basics" >}}{{< /anchor >}}Lab Basics

{{< tableopen >}}{{< theadopen >}}{{< tropen >}}{{< thopen >}}
WORKING IN THE LAB
{{< thclose >}}{{< thopen >}}
COMMUNICATING YOUR WORK
{{< thclose >}}{{< trclose >}}{{< theadclose >}}{{< tbodyopen >}}{{< tropen >}}{{< tdopen >}}

{{% resource_link 714642be-45a1-d5f4-6345-5094456059c8 "General lab policies, do's and don'ts" %}}

{{% resource_link 61a035e4-275d-81eb-212e-beb3ab2f7c87 "Guidelines for working in the tissue culture facility" %}}

Guidelines on using personal protective equipment, developed by   
MIT's Environmental Health and Safety Office:

- [Personal Protective Equipment](https://ehs.mit.edu/workplace-safety-program/personal-protective-equipment/)
- Lab coat ([PDF](http://openwetware.org/images/4/47/2010_EHS_LabCoatGuidance.pdf))

{{< tdclose >}}{{< tdopen >}}

{{% resource_link 02782790-19cc-d222-7cb2-c2e7106d46b7 "Statement on Collaboration and Integrity" %}}

{{% resource_link 3b6c19a7-06ec-ef2a-ad75-041f80286fd6 "Guidelines for maintaining your lab notebook" %}}

{{% resource_link c054e983-55c4-7571-51ab-8f1efea6beb0 "Guidelines for oral presentations" %}}

{{% resource_link 4475e656-793c-6dff-b84a-59b0ac66adb2 "Guidelines for writing up your research" %}}

{{< tdclose >}}{{< trclose >}}{{< tbodyclose >}}{{< tableclose >}}

## {{< anchor "Orientation" >}}{{< /anchor >}}First Day: Lab Orientation

There are ten stations for you and your lab partner to visit on your lab tour today. Some will be guided tours with a TA or faculty there to help you and others are self-guided, leaving you and your partner to try things on your own. Your visit to each station should last ~10-15 minutes. It doesn't matter which station you visit first but you must visit them all before you leave today. Afterward, you should have some time remaining to begin the For Next Time assignment, and are strongly encouraged to do so. Your lab practical next time will assess your mastery of a selection of these stations - all are fair game, as is the lab safety material covered in today's pre-lab lecture.

{{< tableopen >}}{{< theadopen >}}{{< tropen >}}{{< thopen >}}
PRE-LAB LECTURE
{{< thclose >}}{{< thopen >}}
LAB Materials
{{< thclose >}}{{< trclose >}}{{< theadclose >}}{{< tbodyopen >}}{{< tropen >}}{{< tdopen >}}
({{% resource_link d00ac979-5def-7fb9-7c67-db6f41d3dc8d "PDF" %}}) 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 84e8174c-592f-f934-08b5-8921e05ba894 "Lab orientation" %}}
{{< tdclose >}}{{< trclose >}}{{< tbodyclose >}}{{< tableclose >}}

## {{< anchor "RNA_Engineering" >}}{{< /anchor >}}Module 1: RNA Engineering

**Instructors**: Jacquin Niles and Agi Stachowiak

In this module you will investigate RNA aptamer selection. You may already be familiar with peptides or proteins, such as antibodies, that bind to specific molecules. Short fragments of RNA – called RNA aptamers - can have secondary structures that also allow them to bind a target molecule with good affinity and specificity. Normally, RNA aptamers that bind particular targets are found by screening many candidates at random in a process called SELEX, or systematic evolution of ligands by exponential enrichment; however, predictive computational tools can also be used. In the coming weeks, you will essentially perform one round of SELEX. Because SELEX typically takes several rounds to isolate target-binding aptamers, you will start with a known aptamer mixture rather than than a completely random library. Your goal will be to explore what experimental parameters affect the enrichment of a heme-binding RNA aptamer from a mixture of heme-binding and non-binding RNAs.

**Acknowledgement**: We thank 20.109 instructor Natalie Kuldell for helpful discussions and for acquiring funding for module development.

{{< resource d6cf9db1-840b-790f-a496-1a103d1291e4 >}}

Public domain image. (Prepared using RNA Folding (mfold) at the [mFold Web Server](http://mfold.rna.albany.edu/?q=mfold/download-mfold)). Reference: Zuker, M. "Mfold Web Server for Nucleic Acid Folding and Hybridization Prediction." _Nucleic Acids Res._ 31, no. 13 (2003): 3406-3415.

{{< tableopen >}}{{< theadopen >}}{{< tropen >}}{{< thopen colspan="3" >}}
{{< anchor "MODUle_1:_RNA_Engineering" >}}{{< /anchor >}}MODUle 1: RNA Engineering
{{< thclose >}}{{< trclose >}}{{< tropen >}}{{< thopen >}}
LAB DAYS
{{< thclose >}}{{< thopen >}}
PRE-LAB LECTURES
{{< thclose >}}{{< thopen >}}
LAB Materials
{{< thclose >}}{{< trclose >}}{{< theadclose >}}{{< tbodyopen >}}{{< tropen >}}{{< tdopen >}}
Day 1
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 97f8b3cb-baed-0a4d-1d8b-30a3df505b77 "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link a4698e5d-1f6b-3e32-71de-c8f8829f224f "Amplify aptamer-encoding DNA" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 2
{{< tdclose >}}{{< tdopen >}}
({{% resource_link f14bd5af-7a9c-5a5a-f852-6261995ba71f "PDF" %}}) 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 52bbd0a9-989c-ea78-b8db-094bc1580980 "Purify aptamer-encoding DNA" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 3
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 12f68f84-e49f-4a3b-8865-de592b4758ad "PDF" %}}) 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 5edcc5b0-04cc-d6d9-c6f5-b93bf4f13870 "Prepare RNA by IVT" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 4
{{< tdclose >}}{{< tdopen >}}
({{% resource_link d58d9787-8bd3-a59f-8f76-fa6ad413c5b8 "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 6bee5b41-a34d-cb2b-7923-c9303daa44b8 "Purify RNA and run affinity column" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 5
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 1cca677f-b8b3-641d-c56a-8d87b58cc038 "PDF" %}}) 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link e11e968b-82aa-fefc-994a-6731a910335d "RNA to DNA by RT-PCR" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 6
{{< tdclose >}}{{< tdopen >}}
({{% resource_link d867998b-5e5a-4fef-939d-99a3b3e23e1a "PDF" %}}) 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link e22f2979-02d0-a74e-610c-22f87f1c3eac "Post-selection IVT and journal club" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 7
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 0542b9b3-a6e7-d1a6-c103-8cd5b2537837 "PDF" %}}) 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 6074c179-64ef-9a51-f5e9-785309daca35 "Aptamer binding assay" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 8
{{< tdclose >}}{{< tdopen >}}
 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 5dbc8036-e0e3-9c3c-1973-b5a70d1971f1 "Journal club (contd.)" %}}
{{< tdclose >}}{{< trclose >}}{{< tbodyclose >}}{{< tableclose >}}

See the {{% resource_link 5a2e497f-e49c-5201-6e67-a52f578758e7 "Assignments" %}} page for descriptions of the Module 1 laboratory report and RNA computational analysis assignments.

{{% resource_link d6824c94-8959-a4ec-b5b3-a6ce9cdcde6d "TA notes for Module 1" %}}

## {{< anchor "Protein_Engineering" >}}{{< /anchor >}}Module 2: Protein Engineering

**Instructors**: Alan Jasanoff and Agi Stachowiak

In this experiment, you will modify a protein called inverse pericam (developed by [Nagai, et al](http://www.ncbi.nlm.nih.gov/pubmed/11248055).) in order to affect its functions as a sensor. Inverse pericam (IPC) comprises a permuted fluorescent protein linked to a calcium sensor. The "inverse" in the name refers to the fact that this protein shines brightly in the absence of calcium, but dimly once calcium is added. The dissociation constant _K_{{< sub "D" >}} of wild-type IPC with respect to calcium is reported to be 0.2 μM (see also figure below). Your goal will be to shift this titration curve or change its steepness by altering one of the calcium binding sites in IPC's calcium sensor portion. You will modify inverse pericam at the gene level using a process called site-directed mutagenesis, express the resultant protein in a bacterial host, and finally purify your mutant protein and assay its calcium-binding activity via fluorescence. In the course of this module, we will consider the benefits and drawbacks of different approaches to protein design, and the types of scientific investigations and applications enabled by fluorescently tagged biological molecules.

**Reference**: Nagai, T., et al. "[Circularly Permuted Green Fluorescent Proteins Engineered to Sense Ca2+](http://dx.doi.org/10.1073/pnas.051636098)." _PNAS_ 98, no. 6 (March 6, 2001): 3197-3202. \[Open Access\]

We gratefully acknowledge 20.109 instructor Natalie Kuldell for helpful discussions during the development of this module, as well as for her prior work in developing a [related module](http://openwetware.org/wiki/20.109:Module_2) in the Spring 2007 course.

{{< resource c54f55f1-3f57-fa02-f1a1-6b41a696bf73 >}}

Raw titration curve for IPC. Shown here is sample data from the teaching lab: normalized fluorescence for wild-type inverse pericam as a function of calcium concentration. As you will later learn, an apparent KD can be estimated from such a plot: it is the point on the x-axis where the curve crosses y = 50%, or ~0.5 μM here

{{< resource 041afc0e-47f2-ad63-d7ff-f6d92b255a6d >}}

Fitted titration curve for IPC. A more sophisticated analysis using curve-fitting reveals KD to be ~ 0.3 μM, closer to the reported value for inverse pericam.

{{< tableopen >}}{{< theadopen >}}{{< tropen >}}{{< thopen colspan="3" >}}
{{< anchor "Module_2:_Protein_Engineering" >}}{{< /anchor >}}Module 2: Protein Engineering
{{< thclose >}}{{< trclose >}}{{< tropen >}}{{< thopen >}}
LAB DAYS
{{< thclose >}}{{< thopen >}}
PRE-LAB LECTURES
{{< thclose >}}{{< thopen >}}
LAB Materials
{{< thclose >}}{{< trclose >}}{{< theadclose >}}{{< tbodyopen >}}{{< tropen >}}{{< tdopen >}}
Day 1
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 1fcae27c-2a5f-c115-0cdd-4e0ce0df1108 "PDF" %}}) 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 8f241b98-5c89-b697-aa6c-acb092a600c2 "Start-up protein engineering" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 2
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 593fce26-341a-7243-4b83-72e3bd206ee8 "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 6750e9c6-1f55-7b6e-d2f5-1eb58c7afcc0 "Site-directed mutagenesis" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 3
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 0c29e002-f4a0-f180-5d87-8a9bd86817d2 "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 080f3ca5-b66b-7f32-2240-25c3a1fd5acc "Bacterial amplification of DNA" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 4
{{< tdclose >}}{{< tdopen >}}
({{% resource_link da87c792-e680-ce95-6683-a3695b600f7b "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link ede1b5d1-f140-0aec-e47a-e3cdc1a2e24f "Prepare expression system" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 5
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 9da00fbd-694a-0173-a696-1b9868938b75 "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link c902c8ea-545c-743f-443e-440ff96d6753 "Induce protein and evaluate DNA" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 6
{{< tdclose >}}{{< tdopen >}}
({{% resource_link bf7f22b7-bcb3-0399-8201-2d01870cb35f "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 9c1357cd-6c40-9ae7-8aaa-ac8a92d65cee "Characterize protein expression" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 7
{{< tdclose >}}{{< tdopen >}}
 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 7e3077c4-fdbc-7693-27b3-9a99e5bf2125 "Assay protein behavior" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 8
{{< tdclose >}}{{< tdopen >}}
 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link d9bfa075-4ac8-ea99-8914-f82c9edfb492 "Data analysis" %}}
{{< tdclose >}}{{< trclose >}}{{< tbodyclose >}}{{< tableclose >}}

See the {{% resource_link 5a2e497f-e49c-5201-6e67-a52f578758e7 "Assignments" %}} page for a description of the Module 2 protein engineering research paper.

{{% resource_link 8e1e3a8e-c2e0-ebd7-45be-789770a7c53b "TA notes for Module 2" %}}

## {{< anchor "Biomaterial_Engineering" >}}{{< /anchor >}}Module 3: Cell-Biomaterial Engineering

**Instructor**: Agi Stachowiak

_What makes a cell become one type and not another? How can we influence this process, and why would we even want to? When faced with conflicting information – in our own experiments, or in the broader scientific literature – how do we determine what is credible?_ These are just some of the questions you will explore in the third and final module, all in the context of tissue engineering. The goal of tissue engineering (also called regenerative medicine) is to repair tissues damaged by acute trauma or disease. Repair is stimulated by insertion of a porous scaffold at the wound or disease site; the scaffold may carry relevant mature or progenitor cells, and in some cases also soluble growth factors. In cartilage tissue, mature cells are called chondrocytes, and their progenitor cells are mesenchymal stem cells. Tissue regeneration shares many characteristics with natural tissue development, including the importance of appropriate cell differentiation and phenotype maintenance. You will perform a hypothesis-driven investigation of the effects of environmental manipulations on primary chondrocytes and/or mesenchymal stem cells. In particular, you will assess cell viability, genotype, and protein production, but the specific experimental question is up to you.

I gratefully acknowledge Professor Alan Grodzinsky and several members of his lab group (particularly Rachel Miller and Paul Kopesky), for their technical advice and stimulating discussions during the development of this module.

{{< resource 0201cdef-b5d0-bbcc-bcb5-e27b697090f4 >}}

Morphology of primary bovine chondrocytes grown under two different culture conditions. Optical micrographs of chondrocytes grown in monolayer (left) and alginate bead culture (right) are shown. Cells in the 3D culture retain a round phenotype, while cells on the flat surface extend processes and spread out.

{{< tableopen >}}{{< theadopen >}}{{< tropen >}}{{< thopen colspan="3" >}}
{{< anchor "Module_3:_Cell-Biomaterial_Engineering" >}}{{< /anchor >}}Module 3: Cell-Biomaterial Engineering
{{< thclose >}}{{< trclose >}}{{< tropen >}}{{< thopen >}}
Lab DAYS
{{< thclose >}}{{< thopen >}}
PRE-LAB LECTURES
{{< thclose >}}{{< thopen >}}
LAB Materials
{{< thclose >}}{{< trclose >}}{{< theadclose >}}{{< tbodyopen >}}{{< tropen >}}{{< tdopen >}}
Day 1
{{< tdclose >}}{{< tdopen >}}
({{% resource_link a28729ba-6bde-8f6b-5f54-2870adcf37c7 "PDF)" %}}
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 5eaf324b-de14-6e78-7a5e-edddab8e6498 "Start-up biomaterials engineering" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 2
{{< tdclose >}}{{< tdopen >}}
 
{{< tdclose >}}{{< tdopen >}}
{{% resource_link b802fd25-d874-3abb-5ebe-1fab28e947bd "Initiate cell culture" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 3
{{< tdclose >}}{{< tdopen >}}
({{% resource_link c13e01d5-3532-b606-bafe-6ae45b5bbb00 "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link a6ef5fb5-bb91-ae11-a408-6647172e10b5 "Testing cell viability" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 4
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 7efe1a3f-9fcc-aa40-a3e5-1f51100ca1b1 "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 9f1add66-0448-ffba-794a-48d70e4c97b2 "Preparing cells for analysis" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 5
{{< tdclose >}}{{< tdopen >}}
({{% resource_link 7efe1a3f-9fcc-aa40-a3e5-1f51100ca1b1 "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 19f9a2df-e9b2-ac78-8d63-8c61e8b08b1d "Transcript-level analysis" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 6
{{< tdclose >}}{{< tdopen >}}
({{% resource_link e64434f7-7d3b-c569-f49e-ac5c6dce808b "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link f82c52d5-f03d-5172-9e51-338ebfdc2fc4 "Protein-level analysis" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 7
{{< tdclose >}}{{< tdopen >}}
({{% resource_link e5371e89-e3b8-fc4a-b199-3d1f1f6fa34e "PDF" %}})
{{< tdclose >}}{{< tdopen >}}
{{% resource_link 7e3077c4-fdbc-7693-27b3-9a99e5bf2125 "Wrap-up analysis" %}}
{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
Day 8
{{< tdclose >}}{{< tdopen >}}
 
{{< tdclose >}}{{< tdopen >}}
Student presentations
{{< tdclose >}}{{< trclose >}}{{< tbodyclose >}}{{< tableclose >}}

See the {{% resource_link 5a2e497f-e49c-5201-6e67-a52f578758e7 "Assignments" %}} page for a description of the Module 3 cell-biomaterial engineering report.

{{% resource_link 6b1bae8b-ac67-d9ae-c31a-bac7f34d1632 "TA notes for Module 3" %}}